openeogeotrellis/config/config.py

```diff
@@ -25,7 +25,7 @@ class GpsBackendConfig(OpenEoBackendConfig):
     allow_run_udf_in_driver: bool = False
 
     # TODO: eliminate hardcoded VITO-specific defaults?
-    logging_es_hosts: str = os.environ.get("LOGGING_ES_HOSTS", "https://es-infra.vgt.vito.be")
+    logging_es_hosts: str = os.environ.get("LOGGING_ES_HOSTS", "https://es-infra.vgt.vito.be").split(",")
```

Comment:
I think the type will have to change from str to list ? Because the split method returns a list of the parts?
May also have impact further on.